src/jit/gtlist.h

```diff
@@ -145,6 +145,10 @@ GTNODE(LT               , "<"            ,GenTreeOp          ,0,GTK_BINOP|GTK_RE
 GTNODE(LE               , "<="           ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
 GTNODE(GE               , ">="           ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
 GTNODE(GT               , ">"            ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
+#ifndef LEGACY_BACKEND
```

Comment:
I'll add some comments. 

I'm not sure about ifdefs, as far as I can tell ARM has a TST instruction that's similar to x64's TEST instruction so these could be used on ARM once codegen adds the necessary support.